src/app/components/Ionic/IonicProvider.tsx

```diff
@@ -0,0 +1,38 @@
+import { createContext, FC, PropsWithChildren, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Capacitor } from '@capacitor/core'
+import { App } from '@capacitor/app'
```

Comment:
I guess it's too late, but is there an option not to add capacitor bundles to web builds for example?

Reply:
It's not too late. I was thinking of dynamically importing @capacitor/app since it is only used in Ionic platform. But will still add to the bundle in the current implementation. But I figured we rather implement some Ionic specific features, and see after how it could be consolidated in the best way. So it doesn't add to bundle size on web and extension. If that sounds fine?